backend/coins/eth/account.go

```diff
@@ -119,7 +119,7 @@ func (account *Account) Info() *accounts.Info {
 			account.signingConfiguration.ScriptType(),
 			account.signingConfiguration.AbsoluteKeypath(),
 			account.signingConfiguration.ExtendedPublicKeys(),
-			account.signingConfiguration.Address(),
+			account.address.String(),
```

Comment:
I thought we wanted to add the address when creating the signing configuration for ETH?

Reply:
Might be the best after all :/

Reply:
@KasparEtter Would that now mean though we would have to switch on the account type in the keystores.go where the signing configuration is created for the accounts?
`return signing.NewConfiguration(`
        `scriptType, absoluteKeypath, extendedPublicKeys, "", signingThreshold), nil`
Since now we would have to create a configuration with an address for ethereum, but without otherwise